charts/minecraft/values.yaml

```
@@ -18,6 +18,10 @@ resources:
     memory: 512Mi
     cpu: 500m
 
+lifecycle:
+  postStart: {}
+  preStop: {}
```

Shouldn't these be lists?

```suggestion
  postStart: []
  preStop: []
```

---

Sure they should. Now it must be fixed